pkg/proxy/winkernel/proxier.go

```diff
@@ -563,7 +565,7 @@ func NewProxier(
 
 	// Network could have been detected before Remote Subnet Routes are applied or ManagementIP is updated
 	// Sleep and update the network to include new information
-	if hnsNetworkInfo.networkType == "Overlay" {
+	if hnsNetworkInfo.networkType == NETWORK_TYPE_OVERLAY {
```

Comment:
could you switch `==` to `strings.EqualFold()` for all places?

Reply:
done.